osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -25,7 +25,7 @@ namespace osu.Game.Rulesets.Catch.UI
 {
     public class CatcherArea : Container
     {
-        public const float CATCHER_SIZE = 100;
+        public const float CATCHER_SIZE = 106.75f;
```

Comment:
Have you visually tested this change? I thought I had it pretty much perfect in https://github.com/ppy/osu/pull/3466 , wanna provide a screenshot in a similar fashion?

Reply:
I haven't visually tested this yet, I reverse engineered the value from stable and plugged it in, and this fixed difficulty calculation (it is now the same as stable). I'll take some screenshots comparing it to stable when I get home.

Reply:
I took a screenshot on stable, latest lazer release and my branch, on a test map using the default skin.

Stable| Lazer release | This PR
--- | --- | ---
![](https://i.imgur.com/GHeC4wz.png) | ![](https://i.imgur.com/b4rXwTL.png) | ![](https://i.imgur.com/Wv0SQx4.png)

However, the playfield in lazer is bigger than it is in stable. Here's stable scaled up by about 7.5% and this PR overlayed:
![](https://i.imgur.com/IuW9wuV.png)

Reply:
I see, that looks good to me.